backend/src/main/java/org/eclipse/tractusx/puris/backend/stock/logic/adapter/ItemStockSammMapper.java

```diff
@@ -121,13 +121,17 @@ public List<ReportedProductItemStock> itemStockSammToReportedProductItemStock(It
             log.warn("Direction should be INBOUND, aborting");
             return outputList;
         }
-        var mpr = mprService.findByPartnerAndPartnerCXNumber(partner, matNbrCatenaX);
 
-        if (mpr == null) {
+        Material material = materialService.findByMaterialNumberCx(matNbrCatenaX);
+        if (material == null) {
             log.warn("Could not identify materialPartnerRelation with matNbrCatenaX " + matNbrCatenaX + " and partner bpnl " + partner.getBpnl());
             return outputList;
         }
 
+        var mpr = mprService.find(material, partner);
+
+
```

Comment:
please remove multiple blank lines, same for lines 181,182.

Reply:
Done